open_prices/prices/tests.py

```diff
@@ -108,11 +108,20 @@ def test_price_without_product_validation(self):
             price=3,
             price_per=price_constants.PRICE_PER_KILOGRAM,
         )
-        self.assertRaises(
-            ValidationError,
-            PriceFactory,
+        with self.assertRaises(ValidationError) as cm:
+            PriceFactory(
+                product_code=None,
+                category_tag="test",
+                price=3,
+                price_per=price_constants.PRICE_PER_KILOGRAM,
+            )
+        self.assertEqual(
+            cm.exception.messages[0],
+            "Invalid value: 'test', expected value to be in 'lang:tag' format",
+        )
+        PriceFactory(
             product_code=None,
-            category_tag="test",
+            category_tag="fr: Grenoble",  # valid (even if not in the taxonomy)
```

Comment:
@raphael0202 I added a test. it passes. should it ?

Reply:
Yes it should! That's one of the major change, as long as the category is language prefixed properly, we accept it as a valid value.